scripts/importing-Exporting-Config.js

```diff
@@ -0,0 +1,508 @@
+function setExportLink(addend)
+{
+    $('#configExportLink').attr('href', 'data:text/plain;charset=utf-8,' + addend);
+}
+
+function exportData()
+{
+    setExportLink(''); //default the link
+    var config_json = encodeURIComponent(getExportData());
+    setExportLink(config_json);
+}
+
+function getExportData() //returns prettified (tabbed) json.stringify version of current config
+{
+    var checkedSeasonList = getCurrCheckedSeasons().split(','); // csv string --> array
+
+    var output = {};
+    output['Seasons in this configuration'] = checkedSeasonList; //sets season list at top of config
+
+    for (var i = 0; i < checkedSeasonList.length; i++) //loop through each checked season
+    {
+        var season = checkedSeasonList[i];
+
+        var cookieList = document.cookie.split(';');
+
+        var elementsArr = []; //for elements of a season
+        var questionsArr = []; //for pit questions of a season
+        var eventsArr = []; //for events of a season (output will include teams going to each event nested under each event)
+        var teamsInEventsArr = [];
+        var teamsArr = []; //for team data of a season
+        
+        var elementStart = `/season_config/${season}/`;
+        var pitStart = `${season} pitQuestions`;
+        var eventStart = `${season} events`;
+        
+        for (var j = 0; j < cookieList.length; j++)
+        {
+            var cookie = cookieList[j].trim();
+            var cvalue = cookie.substring(cookie.indexOf('=')+1);
+            
+            if (cookie.startsWith(elementStart)) //if cookie contains an element
+            {
+                var objName = getElementName(cookie);
+                var objValue = getElementValue(cookie);
+                var pair = {};
+                pair[objName] = objValue;
+                elementsArr.push(pair);
+            }
+
+            if (cookie.startsWith(pitStart)) //if cookie contains the pit question list
+            {
+                //loop through each question in cookie and add that to questionsArr
+                var questionList = cvalue.split(COOKIE_QUESTION_SEPARATOR).slice(0,-1); //slice to take off the last '' in arr
+                for (var k = 0; k < questionList.length; k++)
+                {
+                    questionsArr.push(questionList[k]);
+                }
+            }
+
+            if (cookie.startsWith(eventStart)) //adding events to a season and all teams going to that event
+            {
+                var eventList = cvalue.split(EVENT_LIST_COOKIE_SEPARATOR).slice(0,-1); //slice takes off last '' in arr
+                for (var k = 0; k < eventList.length; k++)
+                {
+                    //add to event list arr
+                    var currEvent = eventList[k];
+                    eventsArr.push(currEvent);
+                }
+            }
+        }
+
+        // looping through cookies for each season \/ \/
+        var elementsObjName = season + ' elements';
```

Comment:
and what if season contains ' elements' in it? or ' events'?

Just a thought, but would it be beneficial to store in a slightly different format like:
```json
{
  "{Season 1}":{
   "elements": elementsArr,
  "pit Questions": questionsArr,
  "Events": eventsArr,
  "TeamData": returnTeamDataArr(...)
  },
  "{Season 2}":{
  ...
 }
}
```